### PR TITLE
Add peso symbol to constants

### PR DESCRIPTION
**Pull request by jdebacker**
This PR updates `constants.py` to include the ₱ symbol.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.0.6] - 2024-12-06 11:00:00
```
Comment (rickecon) [inline CHANGELOG.md:9]: @jdebacker. Update this to 0.0.7.

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ["3.10", "3.11"]
+        python-version: ["3.10", "3.11", "3.12"]
```
Comment (rickecon) [inline .github/workflows/build_and_test.yml:27]: @jdebacker. This line should change to be just `"3.11", "3.12"`.

File: .github/workflows/build_and_test.yml [hunk 3]
```diff
@@ -35,9 +35,9 @@ jobs:
       - name: Setup Miniconda using Python ${{ matrix.python-version }}
         uses: conda-incubator/setup-miniconda@v3
         with:
-          miniforge-variant: Mambaforge
+          miniconda-version: "latest"
           auto-update-conda: true
-          activate-environment: ogphl-dev
+          activate-environment: ogcore-dev
```
Comment (rickecon) [inline .github/workflows/build_and_test.yml:40]: @jdebacker. This line should be `ogphl-dev`

File: docs/book/content/contributing/contributor_guide.md [hunk 4]
```diff
@@ -163,6 +163,6 @@ situations, in which case other contributors are here to help.
 (Sec_ContribFootnotes)=
 ## Footnotes
 
-[^recent_python]:The most recent version of Python from Anaconda is Python 3.12. `OG-PHL` is currently tested to run on Python 3.10 and 3.11.
+[^recent_python]:The most recent version of Python from Anaconda is Python 3.12. `OG-PHL` is currently tested to run on Python 3.10, 3.11, and 3.12.
```
Comment (rickecon) [inline docs/book/content/contributing/contributor_guide.md:166]: @jdebacker. We should remove 3.10 from this line.

File: examples/run_og_phl.py [hunk 5]
```diff
@@ -24,7 +24,9 @@
 def main():
     # Define parameters to use for multiprocessing
     num_workers = min(multiprocessing.cpu_count(), 7)
-    client = Client(n_workers=num_workers, threads_per_worker=1)
+    client = Client(
```
Comment (rickecon) [inline examples/run_og_phl.py:27]: @jdebacker. I don't think we want "`processes=False`" here.

**Comment by rickecon** [comment]
@jdebacker. One other update is to change line 14 of `pyproject.toml` to `target-version = ["py311", "py312"]`.

**Comment by jdebacker** [comment]
@rickecon Thanks for the thorough review.  I've updated the changes to incorporate your suggestions.

**Review by rickecon — APPROVED** [review]

**Comment by rickecon** [comment]
@jdebacker. Thanks for this PR. Looks great. Merging now.

**Pull request merged**